docs/faq.rst

```diff
@@ -147,10 +147,21 @@ simple dictionary.
 
 .. code:: python
 
+    def create_dag(dag_id):
+        """
+        A function returning a DAG object.
+        """
+
+        return DAG(dag_id)
+
+
     for i in range(10):
         dag_id = 'foo_{}'.format(i)
         globals()[dag_id] = DAG(dag_id)
+
         # or better, call a function that returns a DAG object!
+        other_dag_id = 'dag_id_{}'.format('returned')
```

Comment:
```suggestion
        other_dag_id = 'bar_{}'.format(i)
```
dag id must be unique.

Reply:
Apologies for the late response, been away on a long holiday and after had loads on my plate.